setup.py

```diff
@@ -80,6 +80,8 @@
         "ethpm-types>=0.1.0b5",
         "hexbytes>=0.2.2,<1.0.0",
         "packaging>=20.9,<21.0",
+        "pandas>=1.4.0,<2.0",
+        "pandas-stubs>=1.2.0,<2.0",
```

Comment:
are the stubs only used for type-checking? If yes, they can be moved to the `"lint"` section (line 20)